scripts/az_login.sh

```diff
@@ -29,12 +29,6 @@ TENANT_ID=$(az account show --query tenantId -o tsv)
 if [ -z "$SUB_ID" ]; then
   echo -e "\n\e[31m»»» ⚠️ Unable to find Azure subscription."
   exit 1
-elif [ "$SUB_ID" != "$ARM_SUBSCRIPTION_ID" ]; then
```

Comment:
Didn't see a reason why we need to set this locally when it's derived from the az cli context - I added this a while back for convenience but since we don't need the subscription id in config any more for anything else, seemed redundant. We still show the current sub being used which I think is enough

Reply:
as someone who's nearly deployed into prod subscription I'd prefer if it stayed 🤷🏼